Sources/AppKitNavigation/Internal/ErrorMechanism.swift

```diff
@@ -0,0 +1,20 @@
+#if canImport(AppKit) && !targetEnvironment(macCatalyst)
```

Comment:
(This file can move to SwiftNavigation and use the `package` modifier, as well.)